.github/workflows/scripts/unique_warning_reporter.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This file determines and counts the unique warnings that appear in the test suite using the
+# warnings-as-errors action runs. The volume of each reported warning can be used to indicate
+# severity or importance for rectification.
+
+JOBID=$(gh run list -w "Test-suite with Python warnings as errors" -L 1 --json databaseId -q '.[0].databaseId')
```

Comment:
Just for the sake of being verbose
```suggestion
JOBID=$(gh run list -w "Test-suite with Python warnings as errors" -L 1 --json databaseId -q '.[0].databaseId')
echo "View latest job at https://github.com/PennyLaneAI/pennylane/actions/runs/$JOBID"
```

Reply:
Sure, no harm doing this.